gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/header-dialog/header-dialog.component.scss

```diff
@@ -0,0 +1,3 @@
+:host ::ng-deep .mat-dialog-actions {
```

Comment:
not needed ng-deep. you can use: 
![image](https://github.com/gravitee-io/gravitee-api-management/assets/4974420/32032df1-0ff3-4f32-919a-1011ecc3f278)

I think all ng-deep should have a special review and should always be looked at if they can't be avoided.

Reply:
no longer in use